packages/copy-config/package.json

```diff
@@ -1,7 +1,5 @@
 {
-  "bin": {
-    "copy-config": "src/main/cli.js"
```

Comment:
are you sure about this change? Sounds like a breaking change to me,  doesn't it?

Reply:
good catch, this one was not on purpose